controllers/cluster-registration/controller.go

```diff
@@ -1,6 +1,6 @@
 // Copyright Red Hat
 
-package manager
+package registeredcluster
```

Comment:
What will this mean when we ultimately add a second controller for a Compute CR?

Reply:
We will have another directory next to cluster-registration for that controller

Reply:
It sounds like you're then suggesting a manager per controller, which isn't the pattern we've used in other operators. I'm ok moving forward but we may need to revisit this later.